examples/conf/src/data/sponsors.yml

```diff
@@ -0,0 +1,4 @@
+- id: gatsby
```

Comment:
We'll prolly want some type of "tiering" since that seems to be standard at conferences.

Reply:
Yeah, was wondering about that. Maybe just add a `tier` field?